spec/unit/util/tagging_spec.rb

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env ruby
+# coding: utf-8
```

Comment:
Was this added intentionally?

Reply:
my editor did it automatically because it detected something non-ascii in this file